zipline/pipeline/loaders/blaze/core.py

```diff
@@ -952,6 +954,17 @@ def collect_expr(e):
                 columns=added_query_fields + list(map(getname, columns)),
             )
         )
+        if not materialized_deltas.empty and have_sids:
+            missing_sids = materialized_deltas[
+                ~materialized_deltas.sid.isin(assets)
+            ]
+            log.debug(
+                "Didn't find the following sids in asset index: {}".format(
```

Comment:
Ok. Maybe there should be another place (or just separate analysis entirely) to find the missing sids. I can get rid of the logging; it'll simplify things quite a bit.